### ISSUE TITLE
Displayed contents of index.html

### ISSUE THREAD

**Issue by stla** [ramnathv/poirot#17]
Here is my first attempt to use `poirot`: http://stla.github.io/Test_Poirot3/index.html
Please Ramnath feel my gratitude, this is really great. I have published a post containing `rgl` interactive graphics directly, without any bare-hand actions other than running `blogify()` and synchronizing on GitHub :-)

I wonder about what are we supposed to display on the welcome page (index.html). Are we supposed to keep the topleft side panel about `poirot` ? - I don't see how to remove the "fork me on github" link image without modifying index.html, and I have changed `logo: assets/img/poirot.jpg` in `site.yml` but my image does not appear. I also wonder whether we are supposed to keep the "disqus" links. 

And how could we modifiy the style of the welcome page ? For instance I'd like to have more empty space between the links to my articles and the disqus links.

**Comment by ramnathv** [comment]
The short answer is you will have to modify the layout files in `libraries/frameworks/purus/layouts`. I will post an update with the exact modification you need to make to have more empty space. The idea is to use HTML and CSS.

**Comment by stla** [comment]
Thanks. I think you mean `assets\layouts`. I see the link to "fork me on github" in this folder. But about my logo image, I still don't understand why it is not displayed.

**Comment by stla** [comment]
Hey ! My logo image is displayed now. Is it only a question of patience ?

**Comment by ramnathv** [comment]
**Logo**

I see that the image does appear now on the sidebar. I checked the layout and it is linked to `site.logo`, so should work.

**Sidebar**

The sidebar is controlled by this [partial](https://github.com/ramnathv/poirotBlog/blob/gh-pages/libraries/frameworks/purus/partials/sidebar.html). In particular, the image and the forkme button are controlled by this part of the partial

``` html
{{# page.github }}
    <a href="https://github.com/{{user}}/{{repo}}">
      <img style="position: absolute; top: 0; left: 100px; border: 0; height: 100px; z-index: 1;" 
        alt="Fork me on GitHub" 
        src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png" >
    </a>
 {{/ page.github }}
```

The idea is that it uses the properties specified on each page. So if your page has properties

```
---
user: ramnathv
repo: poirot
---
```

then, it will use those properties while rendering. Now, I realize that it is better of making these properties global. So, if you replace `page.github`, with `site.github`, then all properties will be based off the values you enter in `site.yml`. So you could specify the repo and username in one place and have it work seamlessly.

Let me know if you have trouble doing this. You will find the html layout for the sidebar in `libraries/frameworks/purus/partials/sidebar.html`.

**Empty Space**

Here is the relevant [layout](https://github.com/ramnathv/poirotBlog/blob/gh-pages/libraries/frameworks/purus/layouts/deck.html#L26-L27).

``` html
 {{{page.content}}}
 <div id="disqus_thread"></div>
```

 The easiest way to move the disqus comments to the bottom of the page is to write css.

``` css
#disqus_thread {
    margin-top: 40px;
}
```

You can either add this inline or drop it into the layout directly.

**Comment by stla** [comment]
Ah ok so you meant `libraries/frameworks/purus`. I will soon try your hints, thank you very much. And I will migrate my [current blog](http://stla.overblog.com/) to GitHub thanks to `poirot` :) (and thanks to [GitHub for Windows](http://windows.github.com/)).

**Comment by stla** [comment]
I have another question. In order to "embed" a blog article in the R-bloggers site, the R-bloggers webmaster requires the "feed" of the post. I don't know exactly what is it, but more importantly I don't know whether this feed is available with `poirot` on GitHub ?

**Comment by ramnathv** [comment]
I know exactly what you are talking about. I am working on being able to create the XML feed required by R-Bloggers  automatically for Poirot blogs. The idea is to use a mustache layout corresponding to the XML feed format. Now that someone is using Poirot, I have more motivation to do this :) I will update you once I have it set up and working.

**Comment by stla** [comment]
Mmmm ok... so, whenever I build my true blog, maybe the first article I'll publish should be about how I became a `poirot` user. I'm thinking about that.

**Comment by stla** [comment]
Hello Ramnath. Now I have tried your instructions about the sidebar and the empty space, and everything works very well (http://stla.github.io/Test_Poirot3/).

**Comment by ramnathv** [comment]
Looking good. You might want to change the Disqus Comments shortname. I would recommend doing the following
1. Change `page.disqus.shortname` to `site.disqus.shortname` [here](https://github.com/stla/Test_Poirot3/blob/gh-pages/libraries/widgets/disqus/disqus.html#L3)
2. In `site.yml`, add `disqus: {shortname: "your-disqus-short-name"}`

**Comment by stla** [comment]
I have just tried this but now disqus doesn't work anymore: _We were unable to load Disqus. If you are a moderator please see our troubleshooting guide_.

**Comment by ramnathv** [comment]
Do you have a disqus account?

Send me a link to your HTML

**Comment by stla** [comment]
Ahah no ! I don't know yet what is disqus exactly. And actually when you suggested me to change the shortname, I didn't understand what is the role of this shortname.

**Comment by ramnathv** [comment]
[Disqus](http://disqus.com/) allows you to add a commenting mechanism to your web sites. You will need to create an account and then choose a shortname for your blog, which you can then enter in `site.yml`.

**Comment by stla** [comment]
Ok. I have just created an account and put my shortname in `site.yml`, but that still doesn't work. In order to create the shortname I had to register my site, and I've typed http://stla.github.io/Test_Poirot3/

**Comment by ramnathv** [comment]
You need a shortname for every site you create. You will have to use the disqus website to get it. I see from the site  that you have `shortname` as `whereisyourshortname`, which seems like you have just put a placeholder.

You can google for this to figure out how to get a shortname.

**Comment by stla** [comment]
`whereisyourshortname` is what I wrote before creating my account. But now I have written my true shortname : https://github.com/stla/Test_Poirot3/blob/gh-pages/site.yml

**Comment by ramnathv** [comment]
I see that your html files still have `whereisyourshortname` which probably implies that your blog has not been updated. You should run `blogify` again to ensure that the correct shortname gets picked up. I checked your commit history and see that you did not run `blogify` after you fixed the shortname.

**Comment by stla** [comment]
You're right. It works after blogifying. Thank you.

**Comment by ramnathv** [comment]
Great. There are lot of things to be optimized in the workflow. So, feel free to post any feedback/suggestions you have.

**Comment by stla** [comment]
Thank you. I'm taking a break now. My next step will be to try to change some styles. Globally, I prefer the classic knitr style than the current style on my blog (compare for example http://stla.overblog.com/the-binary-splitting-with-the-r-gmp-package-application-to-gauss-hypergeometric-function and http://stla.github.io/Test_Poirot3/posts/BS_F21_v3.html).

**Comment by stla** [comment]
Hello Ramnath,
Have you made some progress for the XML feed ?

**Comment by ramnathv** [comment]
@stla No. I haven't made progress on this. I need a full working session with Poirot to crank these features out. I shall keep you posted.
